assets/js/components/view-switcher/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { ButtonGroup, Button } from '@wordpress/components';
+import { useState, Fragment } from '@wordpress/element';
+import withComponentId from '@woocommerce/base-hocs/with-component-id';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+
+const ViewSwitcher = ( {
+	className,
+	label = __( 'View', 'woo-gutenberg-products-block' ),
+	views = [],
```

Comment:
The default for these props that are now required can be removed. :slightly_smiling_face: